src/main/java/ladysnake/requiem/common/remnant/BasePossessionHandlers.java

```diff
@@ -82,6 +83,12 @@ public static void register() {
             }
             return PossessionStartCallback.Result.PASS;
         });
+        PossessionStartCallback.EVENT.register(Requiem.id("emancipation"), (target, possessor, simulate) -> {
+            if (possessor.hasStatusEffect(RequiemStatusEffects.EMANCIPATION)) {
+                return PossessionStartCallback.Result.ALLOW;
```

Comment:
Wouldn't this allow you to possess anything, without any other addons being able to override this? Possibly unintended/could cause issues

Reply:
Correct ! I added a TODO to remind ourselves to fix this after soul vessels are done.